kubernetes/test-infra/eks/node-config/config.tf

```diff
@@ -16,37 +16,42 @@ locals {
     - system:bootstrappers
     - system:nodes
 MAPPEDROLE
+
 }
 
 resource "local_file" "cluster_ca" {
-  content  = "${base64decode(var.cluster_ca)}"
+  content = base64decode(var.cluster_ca)
   filename = "${path.module}/cluster_ca"
 }
 
 resource "null_resource" "wait_for_api" {
-  depends_on = ["local_file.cluster_ca"]
+  depends_on = [local_file.cluster_ca]
 
   provisioner "local-exec" {
+
+    working_dir = path.module
     command = <<CMDEOF
-while ! curl -s --cacert ${local_file.cluster_ca.filename} ${var.cluster_endpoint}/version > /dev/null; do 
+while ! curl -s --cacert ./cluster_ca ${var.cluster_endpoint}/version > /dev/null; do 
```

Comment:
Is this intended? I think that we're leaking implementation details (i.e. filename) by doing this.

Reply:
The two failed tests are unrelated and were also failing before this change.
I'm working on fixing them next.

![Screenshot 2019-07-25 at 12 23 07](https://user-images.githubusercontent.com/1097371/61867360-4c21dc80-aed7-11e9-91ee-c0d24b5f370e.png)

Reply:
Also, that filename is set on line 24 above, so not exactly obscured :)

Reply:
For now - code can be moved in the future an become obscured, so using references rather than hardcoded duplicated values in this specific case is beneficial IMO and if the previous/existing notation works in 0.12 I don't see a reason why not.

If we break it again in future releases, we should be the first to notice through the dog fooding here, before our users do in the wild 😜

Reply:
Found a solution to re-use the `filename` attribute from `local_file`. Thanks for the tip about `abspath()`.

Reply:
Nice, that's so much cleaner! Thanks.